java/src/jmri/jmrit/beantable/signalmast/SignalMastTableDataModel.java

```diff
@@ -355,6 +382,391 @@ protected boolean matchPropertyName(java.beans.PropertyChangeEvent e) {
         return super.matchPropertyName(e);
     }
 
+    /**
```

Comment:
Since the existing table cell renderer and editor as of #2559 do work, why replace them with this?